cx_Freeze/windist.py

```diff
@@ -373,6 +376,7 @@ def initialize_options(self):
         self.environment_variables = None
         self.data = None
         self.install_icon = None
+        self.all_users = 'False'
```

Comment:
Can you turn this into a real boolean option? Use boolean_options = ["all-users"] on the bdist_msi class. Then you can also just say ``if not self.all_users``.

Reply:
I was not very familiar with bdist_msi class, and wasn't sure that would work. Anyway it is done now!